cfi_forward.adoc

```diff
@@ -75,19 +75,29 @@ The forward-edge CFI introduces the following instructions for landing
 pad operations. All instructions are encoded using the SYSTEM major opcode and
 using the `mop.rr` instructions defined by the Zimop extension.
 
-[width=100%]
-[%header, cols="6,<2,>2,^2,^2,<3,>3,<3,>3,<3,>3,<3,>3", grid=rows, frame=none]
-|===
-|`(mop.rr)`     |`31`  |  `25`|`24`   |`23`   |`22`          |`15`|`14` |  `12`|`11` | `7`|`6` | `0`
-|mnemonic    2+^|`1.00..1`   ^|`t`    |`t` 2+^|`tttsssss`      2+^|`func3`  2+^|`rd`   2+^|`opcode` 
-|`lpsll`     2+^|`1000001`   ^|`0` 3+^| `LLPL`                 2+^|`100`    2+^|`0`    2+^|`1110011` 
-|`lpcll`     2+^|`1000001`   ^|`1` 3+^| `LLPL`                 2+^|`100`    2+^|`0`    2+^|`1110011` 
-|`lpsml`     2+^|`1000011`   ^|`0`    |`0` 2+^|  `MLPL`        2+^|`100`    2+^|`0`    2+^|`1110011` 
-|`lpcml`     2+^|`1000011`   ^|`0`    |`1` 2+^|  `MLPL`        2+^|`100`    2+^|`0`    2+^|`1110011` 
-|`lpsul`     2+^|`1000011`   ^|`1`    |`0` 2+^|  `ULPL`        2+^|`100`    2+^|`0`    2+^|`1110011` 
-|`lpcul`     2+^|`1000011`   ^|`1`    |`1` 2+^|  `ULPL`        2+^|`100`    2+^|`0`    2+^|`1110011` 
-|            2+^|`7`         ^|`1`    |`1` 2+^|  `8`           2+^|`3`      2+^|`5`    2+^|`7` 
-|===
+[wavedrom, , ]
+.... 
+{reg: [
+  {bits:  7, name: 'opcode', attr:'SYSTEM'},
+  {bits:  5, name: 'rd', attr:'0'},
```

Comment:
minor nit: in some places, the 5 bits of 0 are expanded (e.g. sspush rd) and in others it's shown as a single bit (all the lp insts)

Reply:
updated to expanded for all.